js/compiler.ts

```diff
@@ -1,9 +1,8 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 // TODO(ry) Combine this implementation with //deno_typescript/compiler_main.js
 
-/// <reference types="../third_party/node_modules/typescript/lib/typescript.d.ts"/>
-
 import "./globals.ts";
+import "./ts_global.d.ts";
```

Comment:
I guess because you're importing rather than using reference tag, you were able to remove the hack?

Reply:
Yes, so here, we are simply importing a UMD `.d.ts` that imports the `typescript.d.ts` UMD `.d.ts` and augments the global scope.